statsmodels/discrete/tests/test_count_model.py

```diff
@@ -80,8 +80,7 @@ def setup_class(cls):
         cls.res1._results._attach_nullmodel = True
         cls.init_keys = ['exog_infl', 'exposure', 'inflation', 'offset']
         cls.init_kwds = {'inflation': 'logit'}
-        res2 = RandHIE()
-        res2.zero_inflated_poisson_logit()
+        res2 = RandHIE.zero_inflated_poisson_logit
```

Comment:
These seem to change instances to functions.  Is this right?

Reply:
ATM `res2` is a  `RandHIE` instance, and when `zero_inflated_possion_logit` is called, all of `res2`'s relevant attributes are re-set.  After this PR, `zero_inflated_poisson_logit` is just an attribute of `RandHIE`, an object with the same attributes as `res2` in the test.  The PR is changing out a stateful object for a stateless one.

Reply:
Oh -- github was hiding the large diff that changed the results file.  I was looking at the file in master and trying to figure out how this wasn't erroring.  This is "fine" then.